tests/DCPS/XTypes/run_test.pl

```diff
@@ -12,33 +12,44 @@
 $test->enable_console_logging();
 
 my $verbose = 0;
+my $test_name = "";
 GetOptions(
   "verbose" => \$verbose,
+  "test|f=s"   => \$test_name,
 );
 
-my @common_args = ('-DCPSConfigFile', 'rtps_disc.ini');
-push(@common_args, "--verbose") if ($verbose);
+my $common_args = ('-DCPSConfigFile rtps_disc.ini -ORBDebugLevel 1 -DCPSDebugLevel 6');
+if ($verbose) {
+    $common_args = ("$common_args, --verbose");
```

Comment:
```suggestion
  push(@common_args, "--verbose");
```

Reply:
Otherwise the result is `"-DCPSConfigFile rtps_disc.ini -ORBDebugLevel 1 -DCPSDebugLevel 6, --verbose"`